test/commands/diff.test.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable sonarjs/no-duplicate-string */
+import { expect, test } from '@oclif/test';
+
+describe('diff', () => {
+  describe('with file paths, and there are no difference between the files', () => {
```

Comment:
test should be inside `it`

why not changing this:
```
  describe('with file paths, and there are no difference between the files', () => {
    test
      .stderr()
      .stdout()
      .command(['diff', './test/specification.yml', './test/specification.yml'])
      .it('works when file path is passed', (ctx, done) => {
        expect(JSON.stringify(ctx.stdout)).to.equal(
          '"{\\n  \\"changes\\": []\\n}\\n"'
        );
        expect(ctx.stderr).to.equal('');
        done();
      });
  });
```

into this

```
  it('with file paths, and there are no difference between the files', () => {
    test
      .stderr()
      .stdout()
      .command(['diff', './test/specification.yml', './test/specification.yml'])
      
      expect(JSON.stringify(ctx.stdout)).to.equal(
        '"{\\n  \\"changes\\": []\\n}\\n"'
      );
      expect(ctx.stderr).to.equal('');
      done();
      
  });
```

and same for the rest?